src/app/zap-templates/zcl/data-model/chip/color-control-cluster.xml

```diff
@@ -118,63 +118,138 @@ limitations under the License.
       </feature>
     </features>
 
-    <attribute side="server" code="0x0000" define="COLOR_CONTROL_CURRENT_HUE" type="int8u" min="0x00" max="0xFE" writable="false" reportable="true" default="0x00" optional="true">CurrentHue</attribute>
+    <attribute side="server" code="0x0000" define="COLOR_CONTROL_CURRENT_HUE" type="int8u" min="0x00" max="0xFE" writable="false" reportable="true" default="0x00" optional="true">
+        <description>CurrentHue</description>
+        <quality persistence= "nonVolatile" />
```

Comment:
If this is not generated from the spec and can have errors.... Then I would strongly suggest breaking this up into multiple PRs, one per cluster, or something, for review sanity.  Because cross-checking every single bit of this against the spec in one shot is really painful.